debian/changelog

```diff
@@ -1,6 +1,6 @@
 system76-driver (20.04.35~~alpha) focal; urgency=low
 
-  * Daily WIP for 20.04.35
```

Comment:
Have to keep this line but add the next for legacy reasons. make-release will look for this